sycl/test-e2e/ESIMD/unified_memory_api/gather_usm_pvc.cpp

```diff
@@ -0,0 +1,39 @@
+//==------- gather_usm_pvc.cpp - DPC++ ESIMD on-device test ----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-pvc
```

Comment:
Is there any DG2 functionality we can test?

Reply:
gather is supported similarly for DG2 and PVC. I enabled the test for DG2. Fixed in f541a69d936f5c31befd154497f8f1e9f1af7b57